lua/usip_player.lua

```diff
@@ -2260,6 +2287,7 @@ if tArgs.strSecureOptionPhaseTwo ~= strSecureOption then
     strResetVerifySigPath = path.join(tArgs.strSecureOptionPhaseTwo, strNetxName, "verify_sig.bin")
     -- check if the verify_sig file exists
 
+    -- todo why not use get helper file function?
```

Comment:
Probably because it's old code and the paths are also used to check for the files' existence. I might clean that up if I'll do any work on the file checking in the usip player.